vignettes/diving_deeper.Rmd

```diff
@@ -134,44 +136,54 @@ demographics |>
 
 REDCapTidieR supports projects with multiple arms. If you have a project with multiple arms, there will be an additional column `redcap_arm` to identify the arm that the row is associated with.
 
 ## Categorical variables
 
-REDCapTidieR performs a number of opinionated transformations to categorical fields to streamline exploring and working with them.
+REDCapTidieR performs a number of opinionated transformations on categorical fields to streamline exploring and working with them.
 
-Fields that are of type `yesno` or `truefalse` fields are converted to logical variables.
+Fields that are of type `yesno` or `truefalse` are converted to **logical** variables.
 
 ```{r}
 adverse_events$adverse_event_serious |>
   dplyr::glimpse()
 ```
 
-Fields of single-answer categorical type (`dropdown`, `radio`) are converted to factor variables. The levels of the factor and their order are determined by the choices defined in REDCap. This makes it possible to discover all of the different options for a specific field even if not all of those options are represented in the data. For example, even though we only have 5 adverse events in this database, we can see that there are 837 different choices for adverse events.
+Fields of single-answer categorical type (`dropdown`, `radio`) are converted to **factor** variables by default. To understand how REDCapTidieR constructs this factor, consider the Field Editor for the `adverse_event_grade` variable which is a single-answer `radio` field. There are five choices for this field. Choices have **raw values** and **choice labels**, separated by a comma. Here, the raw values are `1`, `2`, `3`, `4`, and `5`, and their associated choice labels are `Grade 1`, `Grade 2`, `Grade 3`, `Grade 4`, and `Grade 5`. We find that choice labels are generally more useful and intuitive to work with than raw values. For this reason, REDCapTidieR derives the labels of the factor from the choice label. The order of the factor levels is the same as the order of choices in REDCap.
+
+<center>
+
+![](field-editor-ae-grade.png)
+
+</center>
+
+The fact that all makes it possible to discover all of the different options for a specific field even if not all of those options are represented in the data. For example, even though we only Grade 1 and 2 adverse events in the database, we can see that there are 5 grades.
 
 ```{r}
-adverse_events$adverse_event_meddra_ae_code
+adverse_events$adverse_event_grade
 ```
 
-In addition, since the order of choices is preserved, it is possible to convert instrinsically ordinal variables to ordinal factors. For example, there is an intrinsic order in adverse event grades.
+Adverse event grades are intrinsically ordered. When a multiple choice field has intrinsically ordered choices, those choices are usually presented in the proper order. Since REDCapTidieR preserves the order of choices from REDCap, you can convert intrinsically ordered variables to ordined factors.
```

Comment:
Is "ordined" a word? Was this meant to be "ordinal factors"?

Reply:
LOL. No. Thanks, fixed :)